internal/command/fn/build.go

```diff
@@ -29,12 +29,31 @@ import (
 )
 
 type Build struct {
-	Name        string `arg:"" help:"the name of the function"`
-	Source      string `arg:"" type:"existingdir" help:"path of the source directory"`
-	Destination string `short:"d" type:"path" help:"path where the compiled wasm file will be saved" default:"."`
-	Language    string `short:"l" enum:"rust,js" required:"" help:"programming language of the function"`
+	Name        string `arg:"" help:"The name of the function"`
+	Source      string `arg:"" type:"existingdir" help:"Path of the source directory"`
+	Destination string `short:"d" type:"path" help:"Path where the compiled wasm file will be saved" default:"."`
+	Language    string `short:"l" enum:"rust,js" required:"" help:"Programming language of the function"`
 }
 
+func (c *Build) Help() string {
+	return `
+DESCRIPTION
```

Comment:
This description needs a little refactor. 

You can change it in: "It compiles a wasm binary from the function specified in the source arg. The "--language" flag is required, with the following possible values: ...."

The rest can be the same but also change "destination directory" to "output directory"